doc/en/components/inputs/select.md

```diff
@@ -0,0 +1,164 @@
+
+```ts
+import { defineComponents,
+         IgcSelectComponent,
+         IgcSelectItemComponent,
+         IgcSelectGroupComponent,
+         IgcSelectHeaderComponent } 
+from 'igniteui-webcomponents';
+
+defineComponents(IgcSelectComponent, IgcSelectItemComponent, IgcSelectGroupComponent, IgcSelectHeaderComponent);
+```
+
+> [!NOTE]
+> Please note that the select header and group components are not mandatory unless you want to use them.
+
+To start using the component add the `igc-select` along with a list of `igc-select-item`'s to choose from:
+
+```html
+<igc-select>
+    <igc-select-item value="Orange">Orange</igc-select-item>
+    <igc-select-item value="Apple">Apple</igc-select-item>
+    <igc-select-item value="Banana">Banana</igc-select-item>
+    <igc-select-item value="Mango">Mango</igc-select-item>
+</igc-select>
+```
+
+### Select 
+
+The `igc-select` component can be used inside and `igc-form` thus it exposes a `Name` property to be registered with. It also has a `Label`, `Placeholder`, and `Size` properties. The `Outlined` property is used for styling purposes only when it comes to the Material theme. Except for the default one, the component provides a few other slots including `header`, `footer`, `helper-text`, `prefix`, `suffix`, and `toggle-icon`. 
+
+### Item
+
+The `igc-select-item` component allows the users to declaratively specify a list of options to be used by the `igc-select` control. Each item provides a `Value` property that represents the data it carries upon selection. The `SelectItem` has a default slot which allows you to specify the content of the item. This content will only be used for visual purposes and will not be used in case the `Value` property is not present on the `igc-select-item`. You could also provide custom content to be rendered before or after the `SelectItem` content using the `prefix` and `suffix` slots. You could predefine a selected item by setting the `Selected` property. You could also disable some or all items via the `Disabled` property.
+
+<code-view style="height: 220px"
+           data-demos-base-url="{environment:dvDemosBaseUrl}"
+           iframe-src="{environment:dvDemosBaseUrl}/inputs/select-item"
+           alt="$Platform$ Select Item Example"
+           github-src="inputs/select/item">
+</code-view>
+
+### Header
+
+You can use the `SelectHeader` to provide a header for a group of items.
+
+<code-view style="height: 250px"
+           data-demos-base-url="{environment:dvDemosBaseUrl}"
+           iframe-src="{environment:dvDemosBaseUrl}/inputs/select-header"
+           alt="$Platform$ Select Header Example"
+           github-src="inputs/select/header">
+</code-view>
+
+### Group
+
+Multiple `igc-select-item`'s can be placed between the opening and close brackets of an `igc-select-group` component so that users can visually group them together. The `igc-select-group` can be labelled via its `label` slot and disabled via its `Disable` property.
```

Comment:
```suggestion
Multiple `igc-select-item`s can be placed between the opening and closing brackets of an `igc-select-group` component so that users can visually group them together. The `igc-select-group` can be labelled via its `label` slot and disabled via its `Disabled` property.
```